installer/roles/local_docker/templates/docker-compose.yml.j2

```diff
@@ -178,7 +178,7 @@ services:
     container_name: awx_postgres
     restart: unless-stopped
     volumes:
-      - "{{ postgres_data_dir }}:/var/lib/postgresql:Z"
```

Comment:
The `12/data` directory must explicitly be bind-mounted to `/var/lib/postgresql/data` where postgresql expects it, else it will init it's own data directory and none of the old data will be in the new AWX install.